### DISCUSSION TITLE
Is this workflow supported?

### DISCUSSION THREAD

**Discussion by kingtat** [Stirling-Tools/Stirling-PDF#938] [General]
I would like to insert Stirling-PDF in my workflow. Scanned documents arrive in an input directory and I move them to sibling directories to be ingested by other applications.
Is it possible to have Stirling, in interactive mode, open files directly from a bind-mount (the input directory above) and save/overwrite those files after editing them in the Stirling UI? This rather than uploading and downloading files over http.

**Comment by Frooodle** [comment]
Either full UI or full folder processing not both 

There is a folder processing logic which there is a .md file you can find to view how to use 

But you can't do folder scanning to then use in UI

**Comment by kingtat** [comment]
Thank you for clarifying that. I'm not sure the sort of ad hoc edits I wish to make can be automated but I will take a look.
I now realise the problem with my workflow stems from the fact I access documents via ssh:// url in my file manager (caja), meaning I cannot drag and drop files concerned on to the the web browser. If I mount the remote directory tree in another way, so drag & drop does work, it will simplify things.
Great tool you have created by the way!
